src/dune_rules/melange/melange_rules.ml

```diff
@@ -287,7 +287,7 @@ let setup_runtime_assets_rules sctx ~dir ~target_dir ~mode
     let loc = mel.loc in
     Memo.parallel_iter copy ~f:(fun (src, dst) ->
         Super_context.add_rule ~loc ~dir ~mode sctx
-          (Action_builder.symlink ~src ~dst))
+          (Action_builder.copy ~src ~dst))
```

Comment:
This solves the problem with JS bundlers. But I am thinking now there are cases where symlinking could be more desirable (e.g. imagine having some melange library with its own `node_modules` folder that includes a few js-only 3rd party libraries that the melange library binds against).

Could we expose the way that runtime_deps copies are created? e.g. `runtime_deps (link ./foo.js)`.

Or maybe what the feature i'm bringing up should be a new feature altogether? 🤔

Reply:
Why is symlinking more desirable in that case? If it's just for performance, I think we should wait for concrete reports of slow builds.

Reply:
Yeah, I was referring to performance. But I agree about testing it first in real world.